docs/fundamentals/read-operations/search-text.txt

```diff
@@ -0,0 +1,157 @@
+.. _laravel-fundamentals-search-text:
+.. _laravel-retrieve-text-search:
```

Comment:
These anchors seem inconsistent with the names in `retrieve.txt`:

```
.. _laravel-fundamentals-retrieve-documents:
.. _laravel-fundamentals-read-retrieve:
```

Is there even a need for multiple anchors at the same location? Presumably, the references to these could just be updated to use the same name.

Reply:
We have some guidelines around moving pages/content - https://wiki.corp.mongodb.com/spaces/DE/pages/201997355/How+To+Add+or+Rename+an+rST+Page+or+Section

That's the reason that this PR contains multiple anchors - preserving the original section headings and adding more targeted ones that match the current file tree structure. We could probably develop better guidelines around the specific words we use in these anchors though!